medical-practitioner-portal/src/app/dashboard/dashboard.component.ts

```diff
@@ -13,10 +18,11 @@ import { MatFormFieldModule } from '@angular/material/form-field';
 import { MatInputModule } from '@angular/material/input';
 import { MatButtonModule } from '@angular/material/button';
 import { MatSelectModule } from '@angular/material/select';
-import { FormsModule } from '@angular/forms';
+import { FormControl, FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { RouterLink, RouterLinkActive } from '@angular/router';
-import {DmerStatusComponent} from '../../../../shared-portal-ui/projects/core-ui/src/lib/case-definitions/dmer-status/dmer-status.component'
-import {DmerTypeComponent} from '../../../../shared-portal-ui/projects/core-ui/src/lib/case-definitions/dmer-type/dmer-type.component'
+import { DmerStatusComponent } from '../../../../shared-portal-ui/projects/core-ui/src/lib/case-definitions/dmer-status/dmer-status.component';
```

Comment:
optional but we can add paths to tsconfig.js e.g. @shared and that would clean up the paths a little

`import { DmerStatusComponent } from '@shared/lib/case-definitions/dmer-status/dmer-status.component';`

Reply:
Just adding @shared is not working so has to use full path to get the shared library working. Tried multiple ways changing the paths for some reason that does not seem to work.